Example/KeyboardSupport/ViewControllerA.swift

```diff
@@ -30,7 +30,8 @@ class ViewControllerA: KeyboardSupportViewController {
         
         let textFields: [UITextField] = [textField1, textField2, textField3, textField4, textField5, textField6, textField7, textField8]
         let keyboardToolbar = KeyboardToolbar()
-        configureKeyboardSupport(with: textFields, scrollView: scrollView, bottomConstraint: nil, usesDismissalView: true, usesKeyboardNextButtons: true, keyboardInputAccessoryView: keyboardToolbar)
+        let configuration = KeyboardSupportConfiguration(textFields: textFields, scrollView: scrollView, usesDismissalView: true, usesKeyboardNextButtons: true, keyboardInputAccessoryView: keyboardToolbar)
```

Comment:
Do usesDismissalView, usesKeyboardNextButton and any optional constraints have reasonable default values that we could use?

Reply:
I think making usesDismissalView and usesKeyboardNextButton have default parameter values of true is more reasonable as most users probably want to have that functionality.